src/main/scala/Pos.scala

```diff
@@ -88,6 +88,18 @@ object Pos {
 
   }
 
+  final case class Go(p: go.Pos) extends Pos {
+
+    val key: String = p.key
+
+    def piotr: Char = p.piotr
+
+    lazy val toInt: Int = (p.file.index << 3) + p.rank.index // todo go change this?
```

Comment:
This looks wrong << 3 is shifting by 3 bits which makes sense if max file size us 8 (2^3). probably wants to be 5 here to allow for 19?